services/snapshotter/service.go

```diff
@@ -16,14 +18,22 @@ import (
 	"github.com/influxdb/influxdb/tsdb"
 )
 
-// MuxHeader is the header byte used for the TCP muxer.
-const MuxHeader = 3
+const (
+	// MuxHeader is the header byte used for the TCP muxer.
+	MuxHeader = 3
+
+	// BackupMagicHeader is the first 8 bytes used to identify and validate
+	// a metastore backup file
+	BackupMagicHeader = 0x59590101
```

Comment:
I know this is already merged, but the 8 byte value of this is `0x0000000059590101`. Did you intend to have this be eight non-zero bytes. Or are the four leading zero bytes intentional? Also, since this is intended to be a fixed size,  would having `BackupMagicHeader` be a typed constant of a fixed size be better? Either `[8]byte` or `uint64`?